lib/core/src/Cardano/Wallet.hs

```diff
@@ -1905,7 +1905,10 @@ mkTxMeta ti' blockHeader wState tx cs expiry =
                 , direction = if amtInps > amtOuts then Outgoing else Incoming
                 , slotNo = blockHeader ^. #slotNo
                 , blockHeight = blockHeader ^. #blockHeight
-                , amount = Quantity $ distance amtInps amtOuts
+                -- fixme: ADP-347 calculate asset balance too
```

Comment:
The `amount` should remain unchanged and ada-only. We will not be providing this information for assets. It is less useful for assets since fees can only be paid in Ada. What assets are being sent can be figured out from the resolved inputs and outputs fully.

Reply:
OK then. Nonetheless I thought it would be good to calculate a total of assets moved in or out of the wallet by the transaction. In `mkTxMeta` it's easier to know which tx outputs are "ours". Resolved inputs and outputs show the address only.

Reply:
This feature is tracked by ADP-683.